apps/emqx/src/emqx_cm.erl

```diff
@@ -334,7 +334,21 @@ takeover_session(ClientId) ->
             takeover_session(ClientId, ChanPid)
     end.
 
-takeover_session(ClientId, ChanPid) when node(ChanPid) == node() ->
+takeover_session(ClientId, Pid) ->
+    try do_takeover_session(ClientId, Pid)
+    catch
+        _ : noproc -> % emqx_ws_connection: call
+            ?tp(debug, "session_gone", #{pid => Pid}),
```

Comment:
Make the kind an atom, so matching messages works faster? Maybe prefix with `persistent_` as well?

Reply:
Copy-paste from previous code. I will remove this since I don't use it in the end. But it affects all sessions, not only persistent sessions.